docs/topics/spider-middleware.rst

```diff
@@ -189,8 +189,10 @@ DepthMiddleware
 .. class:: DepthMiddleware
 
    DepthMiddleware is a scrape middleware used for tracking the depth of each
-   Request inside the site being scraped. It can be used to limit the maximum
-   depth to scrape or things like that.
+   Request inside the site being scraped. It sets `request.meta['depth']`
+   if its not already present in the request and increments it otherwise.
```

Comment:
It'd be nice to say explicitly what value is assigned to `request.meta['depth']` if it is not present (is it 0?). Could you please add it?